packages/docs/docs/communications/organizing-communications.md

```diff
@@ -0,0 +1,480 @@
+# Organizing Communications
+
+# Introduction
+
+Communications in a healthcare context (modeled as [Communication](/docs/api/fhir/resources/communication) resources) can include many scenarios – patient to physician, physician to physician, device to physician, and more. This results in a large volume of communications within this context, so ensuring that they are well organized is vital. For example, a provider may have staff on call to respond to patient messages. Without proper organization, messages may not reach the correct staff member.
+
+This guide covers different ways that you can organize communications and how to think about different properties on the `Communication` element. It will cover:
```

Comment:
> This guide covers different ways that you can organize communications and how to think about different properties on the `Communication` **element**
Be _very_ careful with the terms "resource", "field", and "element"
 - Top level data classes (e.g. `Communication`) are always called "resources"
 - Data attributes on these resources are technically called "elements", though IMO it's ok to use the term "field" as well